packages/core/src/sass/account-wizard.scss

```diff
@@ -385,6 +385,12 @@
             bottom: 0;
             width: 100%;
             padding: 1.5rem;
+            background-color: var(--general-main-1);
+        }
+        @media screen and (max-height: 667px) {
+            .dc-form-submit-button {
+                position: sticky;
+            }
```

Comment:
`@include mobile` is already covering the screen size that you're targeting so no need for media queries here.
```suggestion
            background-color: var(--general-main-1);
            position: sticky;
        }
```

Reply:
@matin-binary the issue with removing media query is , it is then effecting other mobile devices like iphoneXR, pixel 5 etc whose height is greater than 667px , and the button is moving up instead of it's fixed bottom position in live version when compared.

Reply:
My mistake, LGTM